File: Assets/scripts/WuPingControl.js
#pragma strict

function Start () {

}

function Update () {
	transform.position.y -=1*Time.deltaTime;
}
function OnCollisionEnter2D(col:Collision2D){

	if(col.gameObject.name=="hero"){
		gameObject.Find("HeroLife").GetComponent(HeroLife).heroLife+=1;//检测道具与飞机碰撞后添加生命值
		Destroy(this.gameObject);
	}
}

File: Assets/scripts/Cs/WuPingControl.cs
using UnityEngine;
using System.Collections;

public class WuPingControl : MonoBehaviour {

	// Use this for initialization
	void Start () {
	
	}
	
	// Update is called once per frame
	void Update () {
        transform.position = new Vector3(transform.position.x,transform.position.y-Time.deltaTime,transform.position.z);
	}

    void OnCollisionEnter2D(Collision2D col) { 
        if(col.gameObject.name=="hero"){
            GameObject.Find("HeroLife").GetComponent<HeroLife>().heroLife += 1;
            Destroy(this.gameObject);
        }
    }
}
